StyleGuide.md

```diff
@@ -535,52 +540,66 @@ Do not use underlined text in CockroachDB docs. If it seems beneficial to emphas
 
 Whenever a CockroachDB feature is referenced, provide a link to the relevant documentation. You can also provide links to external resources, but only if the resource is confirmed to be accurate by a technical reviewer or the author is a Cockroach Labs SME and no CockroachDB documentation covers the topic.
 
-Avoid using non-descriptive link names such as `here`, `this page`, or `go`.
+Links are marked with inline text surrounded by square brackets followed by the link address in parentheses.
 
-Use Markdown reference-style links when several parts of the same page refer to the same target URL (e.g., [Release Notes](releases/v22.1.html)).
+Avoid using non-descriptive link names such as `here`, `this page`, or `go`.
 
-For websites that automatically localize pages, avoid using localization elements directly within the URL. For example:
+Use Markdown reference-style links when several parts of the same page refer to the same target URL.
```

Comment:
Is it worth providing an example?

Reply:
Yes, I'll add this. I had no idea what reference-style links were before looking them up myself.